nodes/rosa-managing-worker-nodes.adoc

```diff
@@ -13,9 +13,21 @@ ifdef::openshift-rosa[]
 {product-title} (ROSA).
 endif::[]
 
-
 The majority of changes for worker nodes are configured on machine pools. A _machine pool_ is a group of worker nodes in a cluster that have the same configuration, providing ease of management. You can edit the configuration of worker nodes for options such as scaling, instance type, labels, and taints.
 
+ifdef::openshift-dedicated[]
+include::modules/creating-a-machine-pool-ocm.adoc[leveloffset=+1]
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+include::modules/creating-a-machine-pool.adoc[leveloffset=+1]
+include::modules/creating-a-machine-pool-ocm.adoc[leveloffset=+2]
+include::modules/creating-a-machine-pool-cli.adoc[leveloffset=+2]
+
+.Additional resources
+
+* For more information about the available `rosa create machinepool` arguments, see xref:../rosa_cli/rosa-manage-objects-cli.adoc#rosa-create-machinepool_rosa-managing-objects-cli[Managing objects with the rosa CLI].
```

Comment:
The whole page mentioned nothing about taints.

Reply:
Thanks for highlighting that. I will add it to the node label section and make it "Adding node labels and taints".

Reply:
I have now added a new section in this PR on adding taints, both for OSD and ROSA.

I have also created https://github.com/openshift/openshift-docs/pull/43045 to track a requirement to add assemblies on controlling pod placement in the OSD and ROSA docs.I have now added a new section on adding taints, both for OSD and ROSA. I have also created https://github.com/openshift/openshift-docs/pull/43045 to track a requirement to add assemblies on controlling pod placement in the OSD and ROSA docs.I have now added a new section on adding taints, both for OSD and ROSA. I have also created https://github.com/openshift/openshift-docs/pull/43045 to track a requirement to add assemblies on controlling pod placement in the OSD and ROSA docs.